typescript/processors/readTypeScriptModules.js

```diff
@@ -67,27 +68,41 @@ module.exports = function readTypeScriptModules(tsParser, modules, getFileInfo,
           // TODO: find a way of generating docs for them
           if (!resolvedExport.declarations) return;
 
-          var exportDoc = createExportDoc(exportSymbol.name, resolvedExport, moduleDoc, basePath, parseInfo.typeChecker);
+          var exportDoc = createExportDoc(exportSymbol.name, resolvedExport, moduleDoc, basePath, typeChecker);
           log.debug('>>>> EXPORT: ' + exportDoc.name + ' (' + exportDoc.docType + ') from ' + moduleDoc.id);
 
           // Add this export doc to its module doc
           moduleDoc.exports.push(exportDoc);
           docs.push(exportDoc);
 
           exportDoc.members = [];
+          exportDoc.inheritedMembers = [];
           exportDoc.statics = [];
 
           // Generate docs for each of the export's members
           if (resolvedExport.flags & ts.SymbolFlags.HasMembers) {
 
+            var inheritedMembers = resolveInheritedMembers(resolvedExport, typeChecker);
+
+            for (var memberName in inheritedMembers) {
+              if (memberName === 'T') continue;
```

Comment:
I see...
OK well if it worked before then I guess we should leave it the same for consistency for now.
Perhaps we could factor that bit into a shared function so that we only have to change it in one place if we decide it needs fixing in the future?